nodejs/week3/gettingStartedWitheAPI's/src/backend/api/review.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const reviewsRouter = express.Router();
+const dotenv = require('dotenv');
+const mysql = require('mysql');
+dotenv.config();
+const bodyParser = require('body-parser');
+reviewsRouter.use(bodyParser.json());
+
+const { MYSQL_URL } = process.env;
+const connection = mysql.createConnection(MYSQL_URL);
+
+reviewsRouter.get('/', (req, res) => {
+	connection.query('select * from review', (err, results, fields) => {
+		if (err) {
+			return res.send(err);
+		}
+		res.json(results);
+	});
+});
+
+reviewsRouter.post('/', (req, res) => {
+	const review = req.body;
+
+	connection.query('INSERT into review SET ?', review, (err, results, fields) => {
```

Comment:
You doing this a lot. Remember to create a function when you have code that is being copied!

Reply:
Okay. I got it now and created functions for repeated code and made changes to review and reservation file.